pages/login.py

```diff
@@ -0,0 +1,36 @@
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.keys import Keys
+
+# just commenting 
+
+class LoginPage:
+
+    # URL
+
+    URL = 'https://www.saucedemo.com/'
```

Comment:
Base URL would be a better name, and make sure to follow Snake Case naming conventions.